### ISSUE TITLE
Can't run Kivy in jetson nano (Ubuntu 18.04 ARM) with python 3

### ISSUE THREAD

**Issue by solua** [kivy/kivy#6518]
I follow kivy install instruction in python3:
Install virtualenv
Create virtualenv
$ virtualenv -p python3 venv3
Active virtualenv
$ source venv3/bin/activate
Install Kivy
$python -m pip install kivy
But I can't run hello world. I crash when start

> [INFO ] [Logger ] Record log in /home/wee/.kivy/logs/kivy_19-09-14_6.txt
> [INFO ] [Kivy ] v2.0.0.dev0, git-29dfcbc, 20190914
> [INFO ] [Kivy ] Installed at "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/init.py"
> [INFO ] [Python ] v3.6.8 (default, Aug 20 2019, 17:12:48)
> [GCC 8.3.0]
> [INFO ] [Python ] Interpreter at "/home/wee/py3source/pakingkivy/ven3/bin/python"
> [INFO ] [Factory ] 184 symbols loaded
> [INFO ] [Image ] Providers: img_tex, img_dds, img_pil, img_gif (img_pygame, img_ffpyplayer ignored)
> [INFO ] [Text ] Provider: pil(['text_pygame'] ignored)
> Begin run app
> Send message
> brg process start
> brg: Init app
> [CRITICAL] [Window ] Unable to find any valuable Window provider. Please enable debug logging (e.g. add -d if running from the command line, or change the log level in the config) and re-run your app to identify potential causes
> egl_rpi - ImportError: cannot import name 'bcm'
> File "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/core/init.py", line 62, in core_select_lib
> fromlist=[modulename], level=0)
> File "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/core/window/window_egl_rpi.py", line 12, in
> from kivy.lib.vidcore_lite import bcm, egl
> 
> pygame - ModuleNotFoundError: No module named 'pygame'
> File "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/core/init.py", line 62, in core_select_lib
> fromlist=[modulename], level=0)
> File "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/core/window/window_pygame.py", line 13, in
> import pygame
> 
> x11 - ModuleNotFoundError: No module named 'kivy.core.window.window_x11'
> File "/home/wee/py3source/pakingkivy/ven3/lib/python3.6/site-packages/kivy/core/init.py", line 62, in core_select_lib
> fromlist=[modulename], level=0)
> 
> [CRITICAL] [App ] Unable to get a Window, abort.

I repeat step but for python 2.7 in Jetson nano(Ubuntu 18.04 ARM), everything ok

I followed instruction in desktop (Ubuntu 18.04 x86_64) everything fine.

It seem Kivy have problem with python3 in ARM

Would you please help me.
Thanks

**Comment by matham** [comment]
As the error says it, you don't have a window provider, likely because you compiled it incorrectly for the OS or you're missing dependencies. 

Either way, this is not a bug in kivy but a support request, which we do not use the issue tracker for. Please don't open another issue for the problem unless you're explicitly requested to do so in the support channels.

**Comment by solua** [comment]
I confirm Kivy 11.1.1 can run on Jetson nano with python 3.6
It can't be installed from pip wheel but can install manual from source.

https://kivy.org/doc/stable/installation/installation-linux-venvs.html#installation-in-venv

I follow "Dependencies with SDL2" and Kivy work.

Thanks Kivy team for your project.

**Comment by mayureshvs** [comment]
it start giving then segmentation fault to me

**Comment by DBern97** [comment]
> I confirm Kivy 11.1.1 can run on Jetson nano with python 3.6 It can't be installed from pip wheel but can install manual from source.
> 
> https://kivy.org/doc/stable/installation/installation-linux-venvs.html#installation-in-venv
> 
> I follow "Dependencies with SDL2" and Kivy work.
> 
> Thanks Kivy team for your project.

I have just encountered the same issue and installing manually from source also yielded the same issue. However that link does not exist anymore and the installation guide does not really help me to solve the issue.

I post my errors here for clarity, albeit being the same as stated:

`[INFO   ] [Window      ] Provider: sdl2
[DEBUG  ] [Window      ] Ignored <x11> (import error)
[CRITICAL] [Window      ] Unable to find any valuable Window provider. Please enable debug logging (e.g. add -d if running from the command line, or change the log level in the config) and re-run your app to identify potential causes
sdl2 - RuntimeError: b"Couldn't find matching GLX visual"
  File "/usr/lib/python3/dist-packages/kivy/core/__init__.py", line 71, in core_select_lib
    cls = cls()
  File "/usr/lib/python3/dist-packages/kivy/core/window/window_sdl2.py", line 165, in __init__
    super(WindowSDL, self).__init__()
  File "/usr/lib/python3/dist-packages/kivy/core/window/__init__.py", line 1071, in __init__
    self.create_window()
  File "/usr/lib/python3/dist-packages/kivy/core/window/window_sdl2.py", line 315, in create_window
    self.get_gl_backend_name())
  File "kivy/core/window/_window_sdl2.pyx", line 228, in kivy.core.window._window_sdl2._WindowSDL2Storage.setup_window
  File "kivy/core/window/_window_sdl2.pyx", line 76, in kivy.core.window._window_sdl2._WindowSDL2Storage.die

x11 - ModuleNotFoundError: No module named 'kivy.core.window.window_x11'
  File "/usr/lib/python3/dist-packages/kivy/core/__init__.py", line 63, in core_select_lib
    fromlist=[modulename], level=0)
  File "<frozen importlib._bootstrap>", line 1074, in __import__
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked

[CRITICAL] [App         ] Unable to get a Window, abort.
`
plz help
